config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+EFO8HDOzYrkOMazVxontSECWUnM7y6uJsnj+gFaFEBHWH9l5eoCYtOLwTjQadmFVcKgb1ijwdjNhETJxhqVXSz3Xp2MZImE9KUIj6ShBXiFEKB3cJMNEepQsSYTiAAo6EnPA1NfdWhOlf9L3YqzHbmSR0JBLQ+v6NeB94z7bnTZkaUnXsjII7vo+zm4co5EcoZdVRJIGxbJJ2pncKfnBqt2gAbPYiY+fHyPoCweFFxABnopB4MPRgHhd3/2OBnPukQ4/8j3DrqQspWZfSZnI0GIqG1WKgcKhwwPdGuoff2E9P4maQ44GL+1MfGhStEUBzi8qY2Dtec5t8wSHzcEMhDBw/w2xTarZta5X8UA7/ZgDEGKBVostes/Vcj+spocFE9ccZ/ACfhjWjXwr780wx7f7yMLeepNjIB9Y--oG8jfl6RoTwY3vPZ--NfCmNLqGfbgBp+TGBKDLqw==
```

Comment:
Do we actually use this? If not, perhaps it's better to remove it (no config in code)? I'd rather configure secrets through environment variables.

Reply:
This file was created automatically by the `rails new` command. Because we don't use it to save credentials for this app it should be save to delete it.

Reply:
file deleted